apps/src/code-studio/pd/workshop_dashboard/reports/local_summer_workshop_daily_survey/results_loader.jsx

```diff
@@ -57,6 +61,13 @@ export class ResultsLoader extends React.Component {
           facilitatorResponseCounts: data['facilitator_response_counts'],
           courseName: data['course_name']
         });
+
+        if (inExperiment) {
+          this.setState({
+            workshopRollups: data['workshop_rollups'],
+            facilitatorRollups: data['facilitator_rollups']
+          });
+        }
```

Comment:
Nit: It would be slightly preferable here to only make one `setState` call.  In our current version of React (15.x) these two `setState` calls are not batched together, so they trigger separate renders.  Once we upgrade to React 16 I think it won't matter.

This would matter more if the second setState call was somehow dependent on the first, but our pattern here is for them to be independent, so this is a nit.

Related reading:
- https://github.com/facebook/react/issues/10231#issuecomment-316644950
- https://itnext.io/react-setstate-usage-and-gotchas-ac10b4e03d60

Reply:
Although 2 `setState` are independent, both their results should be available before we do rendering. I will fix it to make just 1 `setState` call.